### DISCUSSION TITLE
How to custom getPages redirect to class Livewire?

### DISCUSSION THREAD

**Discussion by phamduc1823** [filamentphp/filament#7973] [General]
how do i click on view that will redirect to class named ViewUser whose namespace is use App\Livewire\ViewUser;
Here is the route:
Route::get('users/{user}', ViewUser::class)->name('view-user');
Here is the ViewResource side:
     public static function getPages(): array
     {
         return [
             'index' => Pages\ListUsers::route('/'),
             'view' => App\Livewire\ViewUser::class, //This is the line I want to access the ViewUser class with the id of that user
         ];
     }

**Comment by leandrocfe** [comment]
what about a custom page?
https://filamentphp.com/docs/3.x/panels/resources/custom-pages#overview

**Reply by phamduc1823** [reply]
thank youuu
